### PR TITLE
chore: update contributing guide

### PR DESCRIPTION
**Pull request by MaikoTan**
I think there are several issues in the previous guide, so I changed the following things:

- Commonly the contributing guide is put into `CONTRIBUTNIG.md` not `.github/contributing.md`.
- Change default branch from `develop` to `master` as `develop` branch is no longer using.
- Use `我们` or `开发团队` to refer to our self.
- Add a code of conduct guide from [Contributor Covenant](https://www.contributor-covenant.org/).

### CODE REVIEW

**Comment by shigma** [comment]
Is there anything else that prevents this PR from being ready for review?

**Comment by MaikoTan** [comment]
> Is there anything else that prevents this PR from being ready for review?

Some sentences are not quite natural, but I think to give it a try is not bad.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+[v2.1]: https://www.contributor-covenant.org/version/2/1/code_of_conduct.html
+[Mozilla CoC]: https://github.com/mozilla/diversity
+[FAQ]: https://www.contributor-covenant.org/faq
+[translations]: https://www.contributor-covenant.org/translations
+
+# 贡献者公约
+
+## 我们的承诺
+
+身为社区成员和贡献者，我们承诺使社区参与者不受骚扰，无论其年龄、体型、可见或不可见的缺陷、族裔、性征、性别认同和表达、经验水平、教育程度、社会与经济地位、国籍、相貌、种族、种姓、肤色、宗教信仰、性倾向或性取向如何。
```
Comment (MaikoTan) [inline CODE_OF_CONDUCT.md:141]:
```suggestion
身为社区成员、贡献者和维护者，我们承诺使社区参与者不受骚扰，无论其年龄、体型、可见或不可见的缺陷、族裔、性征、性别认同和表达、经验水平、教育程度、社会与经济地位、国籍、相貌、种族、种姓、肤色、宗教信仰、性倾向或性取向如何。
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+* 嘲弄、侮辱或诋毁性评论，以及人身或政治攻击
+* 公开或私下的骚扰行为
+* 未经他人明确许可，公布他人的私人信息，如物理或电子邮件地址
+* 其他有理由认定为违反职业操守的不当行为
+
+## 责任和权力
+
+维护者有责任解释和落实我们所认可的行为准则，并妥善公正地对他们认为不当、威胁、冒犯或有害的任何行为采取纠正措施。
+
+维护者有权力和责任删除、编辑或拒绝或拒绝与本行为准则不相符的评论（comment）、提交（commits）、代码、文档（documentation）编辑、议题（issues）或其他贡献，并在适当时机知采取措施的理由。
```
Comment (MaikoTan) [inline CODE_OF_CONDUCT.md:167]:
```suggestion
维护者有权力和责任删除、编辑或拒绝或拒绝与本行为准则不相符的代码、文档、评论（comment）、提交（commits）、议题（issues）或其他贡献，并在适当时机知采取措施的理由。
```

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+
+## Enforcement Responsibilities
+
+Maintainers are responsible for clarifying and enforcing our standards of
+acceptable behavior and will take appropriate and fair corrective action in
+response to any behavior that they deem inappropriate, threatening, offensive,
+or harmful.
+
+Maintainers have the right and responsibility to remove, edit, or reject
+comments, commits, code, documentation, issues, and other contributions that are
```
Comment (MaikoTan) [inline CODE_OF_CONDUCT.md:50]:
```suggestion
Maintainers have the right and responsibility to remove, edit, or reject
code, documentation, comments, commits, issues, and other contributions that are
```

**Review by shigma — APPROVED** [review]

**Pull request merged**